app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/SubmitMessageController.scala

```diff
@@ -67,7 +65,7 @@ class SubmitMessageController @Inject() (
           val (response, movement) = success
```

Comment:
Not sure we need this `response` anymore

Reply:
I don't think we do either, we can look to clean that up